kibble/render/render.go

```diff
@@ -152,6 +175,63 @@ func createLanguage(cfg *models.Config, lang string, locale string) *models.Lang
 		Code:               lang,
 		Locale:             locale,
 		IsDefault:          (lang == cfg.DefaultLanguage),
-		DefinitionFilePath: fmt.Sprintf("%s.all.json", locale),
+		DefinitionFilePath: fmt.Sprintf("%s.all.json", lang),
```

Comment:
I've advocated for [only having the language subtag and not using the regions subtag](https://www.w3.org/International/articles/language-tags/#region) unless needed, e.g. 

> Once again, you should only use region subtags if they are necessary to make a distinction you need. Unless you specifically need to highlight that you are talking about Italian as spoken in Italy you should use it for Italian, and not it-IT. The same goes for any other possible combination.

I'd like to know what other changes we would need to make to not break existing expectations.

Reply:
e.g. underscores to hyphens as well.

Reply:
@stajs works with codes like en and en_AU. However i18n has to be aware of the language so we cant put random langs,  eg, gg_GG does not work. 
@PeculiarGoat thinks we should be doing some validation on these in Uber
currently tracking down a Lang selector bug do to with "es-419"

Reply:
This is too outdated, I don't remember what it was talking about now.